nbbuild/antsrc/org/netbeans/nbbuild/extlibs/DownloadBinaries.java

```diff
@@ -478,6 +472,47 @@ public static MavenCoordinate fromGradleFormat(String gradleFormat) {
             }
             return new MavenCoordinate(group, artifact, version, extension, classifier);
         }
+
+        /**
+         * The maven coordinate is supplied in the form:
+         *
+         * <p>{@code m2:/group:name:version:extension:classifier}</p>
+         *
+         * <p>For the DownloadBinaries task the parts group, name and version
```

Comment:
Not sure what the "canonical" format is, but should extension be required, as in https://github.com/apache/netbeans/blob/master/java/maven.embedder/src/org/netbeans/modules/maven/embedder/impl/MavenProtocolHandler.java ?